### PR TITLE
HTML and CSS file from trail

### PR DESCRIPTION
**Pull request by nilab-khwaja**

### CODE REVIEW

**Comment by nilab-khwaja** [comment]
This is just an exercise to learn how to review code for semantic errors.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
```
Comment (nilab-khwaja) [inline index.html:24]: It would be better to remove this for now, because you didn't have a js script.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <label for ="radio" id="radio1-lable1">Would you recommond our service to a frind</label>
+      <input type="radio" id="radio1" value="yes">
+      <input type="radio" id="radio1" value="No">
+      <br>
+      <input type="textaera" row="5" col="5">
+      <button type="submit" value="submit">submit</button>
+
+    
+
+
```
Comment (nilab-khwaja) [inline index.html:50]: Please remove unwanted space between tags for more readability.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <option value="internet">From the internet</option>
+        
+      </select>
+  
+      <label for ="radio" id="radio1-lable1">Would you recommond our service to a frind</label>
+      <input type="radio" id="radio1" value="yes">
+      <input type="radio" id="radio1" value="No">
+      <br>
+      <input type="textaera" row="5" col="5">
+      <button type="submit" value="submit">submit</button>
```
Comment (nilab-khwaja) [inline index.html:46]: It would be better to enclose your code inside a  div or section to be very organized.

**Review by nilab-khwaja — COMMENTED** [review]
Hi @nilab-khwaja 
Great work until yet but there is little changes need😊.

### **Highlights**
work properly✅
### **Required Changes** ♻️
Check the comments under the review.

**Review by joel-kalema — APPROVED** [review]
Hi @nilab-khwaja 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Your phrases are polishes  👍 
You explained why 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**